package.json

```diff
@@ -13,9 +13,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.5.5",
-    "@chainsafe/eth2.0-config": "^0.1.1",
-    "@chainsafe/eth2.0-types": "^0.1.1",
-    "@chainsafe/ssz": "0.5.2",
+    "@chainsafe/lodestar-config": "^0.5.0",
+    "@chainsafe/lodestar-types": "^0.5.0",
```

Comment:
you can update the lodestar packages to 0.6.0

Reply:
updated to 0.7.0